Overview.src.html

```diff
@@ -3355,119 +3355,84 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 due course.
 <!-- https://lists.w3.org/Archives/Public/public-whatwg-archive/2014Jun/thread.html#msg72 -->
 
-<p>Objects implementing the <code>Body</code> mixin gain an associated
-<dfn title=concept-Body-body>body</dfn> (a byte stream), a
-<dfn title=concept-Body-used-flag>used flag</dfn> (initially unset), and a
+<p>Objects implementing the <code>Body</code> mixin gain a
 <dfn title=concept-Body-MIME-type>MIME type</dfn> (initially the empty byte sequence).
 
+<p>Objects implementing the <code>Body</code> mixin must define an associated
+<dfn title=concept-Body-disturbed>disturbed</dfn> predicate and a
```

Comment:
A predicate is a function that returns a boolean. It seems like a better model than a flag in this case, since you want it to be re-computed every time it is consulted.